pkg/tidb/conn.go

```diff
@@ -28,6 +30,10 @@ import (
 	_ "github.com/jinzhu/gorm/dialects/mysql"
 )
 
+const (
+	EnvTidbOverrideEndpointKey = "TIDB_OVERRIDE_ENDPOINT"
+)
+
 func (f *Forwarder) GetDBConnProps() (host string, port int, err error) {
```

Comment:
prefer to mark this function as private, since it does not provide an overridden address.

Reply:
updated.